artifactory/commands/transferfiles/transfer.go

```diff
@@ -29,8 +29,8 @@ const (
 	uploadChunkSize = 16
 	// Size of the channel where the transfer's go routines write the transfer errors
 	fileWritersChannelSize       = 500000
-	retries                      = 10000
-	retriesWaitMilliSecs         = 5000
+	retries                      = 1800
+	retriesWaitMilliSecs         = 2000
```

Comment:
(2 seconds + 1 second for the REST) X 1800 retries = 1.5 hours. Are we OK with that?